tools/SourceKit/lib/SwiftLang/CodeCompletionOrganizer.h

```diff
@@ -36,7 +36,7 @@ struct Options {
   bool addInnerResults = false;
   bool addInnerOperators = true;
   bool addInitsToTopLevel = false;
-  bool callPatternHeuristics = true;
+  bool callPatternHeuristics = false;
```

Comment:
To keep the behavior of `codecomplete.open` API, this is enabled in `codeCompleteOpen`.